flow/flow.yaml

```diff
@@ -29,27 +29,47 @@ components:
           minItems: 1
           items:
             $ref: './packet-headers/header.yaml#/components/schemas/Flow.Header'
+          x-status:
+            status: deprecated
+            information: |- 
+              Use flow.ingress_packet instead.
           x-field-uid: 2
+        ingress_packet:
+          description: |-
+            Ingress packet
+          type: array
+          minItems: 1
+          items:
+            $ref: './packet-headers/header.yaml#/components/schemas/Flow.Header'
+          x-field-uid: 3
+        egress_packet:
```

Comment:
Egress packet headers continue to have value/valuelist etc. which is not applicable for script. Would be good to have a way  to re-use ingress headers but leave out unused fields from the bundled yaml.